src/pages/Doeun/Login/Components/LoginBox/LoginBox.scss

```
@@ -0,0 +1,143 @@
+.loginBox {
+  width: 350px;
+  min-height: 380px;
+  padding-bottom: 20px;
+  color: black;
+  background-color: white;
+  border: 1px solid #dbdbdb;
+  border-radius: 2px;
+
+  .logo {
+    width: 175px;
+    height: 50px;
+    margin: 22px auto 12px;
+    font-size: 22px;
+  }
+
+  .logo h1 {
+    font-family: instaStyleFont;
+  }
+
+  .login {
+    width: 348px;
+    height: 213px;
+    margin-top: 50px;
+    padding: 0 40px;
+  }
+
+  .login input {
+    width: 100%;
+    height: 38px;
+    margin-bottom: 5px;
+    padding: 10px;
+    background-color: #fafafa;
+    border: 1px solid #dbdbdb;
+    border-radius: 3px;
+    outline: none;
+  }
+
+  .login input.typing {
+    padding-top: 20px;
+    font-size: 12px;
+  }
+
+  .login input:focus {
+    border-color: #a8a8a8;
+  }
```

- 네스팅 + & 연산자 잘 활용해 보세요!
```suggestion
  .login {
     input {
        width: 100%;
        height: 38px;
        margin-bottom: 5px;
        padding: 10px;
        background-color: #fafafa;
        border: 1px solid #dbdbdb;
        border-radius: 3px;
        outline: none;
        
        &.typing {
           padding-top: 20px;
           font-size: 12px;        
        }
        
        &:focus {
           border-color: #a8a8a8;
        }
     }
```

---

반영했습니다~~~!~!